lib/src/mock.dart

```diff
@@ -1242,3 +1242,27 @@ extension ListOfVerificationResult on List<VerificationResult> {
   /// [verifyInOrder].
   List<List<dynamic>> get captured => [...map((result) => result.captured)];
 }
+
+final _defaultDummyValueFor = <Type, dynamic>{
+  num: 0,
+  int: 0,
+  double: 0.0,
+  bool: false,
+  String: '',
+};
```

Comment:
Ideally I'd like these to automatically agree with the dummy values created by the codegen, so maybe codegen should just generate calls to `defaultDummyValue`... but that's not a big deal.